packages/components/src/base-control/index.tsx

```diff
@@ -42,19 +43,21 @@ export { useBaseControlProps } from './hooks';
  * );
  * ```
  */
-export const BaseControl = ( {
-	__nextHasNoMarginBottom = false,
-	id,
-	label,
-	hideLabelFromVision = false,
-	help,
-	className,
-	children,
-}: BaseControlProps ) => {
+const UnconnectedBaseControl = (
+	props: WordPressComponentProps< BaseControlProps, null >
+) => {
+	const {
+		__nextHasNoMarginBottom = false,
+		id,
+		label,
+		hideLabelFromVision = false,
+		help,
+		className,
+		children,
+	} = useContextSystem( props, 'BaseControl' );
+
 	return (
-		<Wrapper
-			className={ classnames( 'components-base-control', className ) }
```

Comment:
Removed this because the context system actually adds the CSS class for us 😳

https://github.com/WordPress/gutenberg/blob/1f9b753b325faaa94107e7e98a4b67132c91e620/packages/components/src/context/use-context-system.js#L53-L56